pinot-core/src/main/java/com/linkedin/pinot/core/query/aggregation/function/AggregationFunctionUtils.java

```diff
@@ -89,4 +90,14 @@ public static String formatValue(@Nonnull Object value) {
       return value.toString();
     }
   }
+
+  public static Serializable getSerializableValue(@Nonnull Object value) {
+    if (value instanceof Long) {
```

Comment:
You don't need to cast here, the cast does nothing.

Also, it would be better to do `if (value instanceof Number)` as Number is the parent class of Long, Double, Integer, etc.